src/Sylius/Bundle/CoreBundle/Announcer/BatchedVariantsUpdateAnnouncer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Announcer;
+
+use Sylius\Component\Product\Command\UpdateBatchedVariants;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+final class BatchedVariantsUpdateAnnouncer implements BatchedVariantsUpdateAnnouncerInterface
+{
+    public function __construct(private MessageBusInterface $messageBus)
+    {
+    }
+
+    public function dispatchVariantsUpdateCommand(array $variants): void
+    {
+        $batchedVariants = array_chunk($variants, 100);
```

Comment:
```suggestion
        $variantsBatches = array_chunk($variants, 100);
```
?

Reply:
I prefer the current one, but it is just personal likeness. Both are fine for me

Reply:
TBH I also prefer my versions :D